docs/configuration/ScenarioDeploymentConfiguration.md

```diff
@@ -330,7 +330,6 @@ Deployment Manager of type `flinkStreaming` has the following configuration opti
 | scenarioStateRequestTimeout                          | duration       | 3 seconds     | Request timeout for fetching scenario state from Flink                                                                                                                                                                                      |
 | jobConfigsCacheSize                                  | int            | 1000          | Maximum number of cached job configuration elements.                                                                                                                                                                                        |
 | miniCluster.config                                   | map of strings | [:]           | Configuration that will be passed to shared `MiniCluster`                                                                                                                                                                                   |
-| miniCluster.streamExecutionEnvConfig                 | map of strings | [:]           | Configuration that will be passed to shared `StreamExecutionEnvironment` used along with  `MiniCluster`                                                                                                                                     |                                                                                                                                                               |
```

Comment:
I realize, that it won't be needed, so I removed it. It is not related to this change.